src/Commands.hs

```diff
@@ -47,57 +47,6 @@ import Types(IOR, REnv(..), getToolConfig, scopeInfo)
 import Util(unpackRelease, fetchDeployContext)
 import Commands.ProxyMode.LocalState(nginxConfTemplate)
 
--- Make the specified release the live release, replacing any existing release.
-select :: T.Text -> IOR ()
-select release = do
-  tcfg <- getToolConfig
-  case tc_deployMode tcfg of
-    DeployMode_select -> selectNoProxy release
-    _ -> P.select release
-
-selectNoProxy :: T.Text -> IOR ()
```

Comment:
I thought we were just going to get rid of the P.select functionality.  The selectNoProxy capability is useful when you want to start a single once in the bootscript for a machine and hence no frontend proxy is required.

Reply:
Looking at your renamings (which look great!), I think that` hx-deploy-tool start` should run the existing "selectNoProxy" code when a proxy is not configured.